src/config/aws.js

```diff
@@ -0,0 +1,19 @@
+const Joi = require('joi')
+
+const expectedAwsEnvironmentValues = {
+  AWS_ACCESS_KEY: Joi.string(),
+  AWS_SECRET_ACCESS_KEY: Joi.string(),
+  AWS_S3_UPDATE_TRANSACTIONS_BUCKET_NAME: Joi.string()
```

Comment:
The AWS SDK has some smart ways of managing credentials so we shouldn't need to include these configs. 

It will use environment variables (in rough order of priority): 

* `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` - in a managed ECS environment this will be used first, the SDK makes a call to an internal AWS service that authorises the instance based on context
* `AWS_ACCESS_KEY_ID` - can be used locally if the session being authorised is for an individual rather than an instance
* `AWS_SECRET_ACCESS_KEY`

Some info here https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-environment.html